From: Qingqing <wangyiqing@kylinos.cn>
Date: Thu, 20 Aug 2026 08:57:39 +0000
Subject: Translated using Weblate (Burmese)

Currently translated at 100.0% (28 of 28 strings)

Translation: openkylin-huanghe/kylin-os-manager-session-service
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/kylin-os-manager-session-service/my/
---
 .../kylin-os-manager-session-service_my.ts         | 168 +++++++++++++++++++++
 1 file changed, 168 insertions(+)
 create mode 100644 daemons/sessionservice/translations/kylin-os-manager-session-service_my.ts

diff --git a/daemons/sessionservice/translations/kylin-os-manager-session-service_my.ts b/daemons/sessionservice/translations/kylin-os-manager-session-service_my.ts
new file mode 100644
index 0000000..4b07e54
--- /dev/null
+++ b/daemons/sessionservice/translations/kylin-os-manager-session-service_my.ts
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="my">
+<context>
+    <name>DiskMonitorExceptionPopup</name>
+    <message>
+        <location filename="../disk_monitor_exception_popup.cpp" line="25"/>
+        <source>Kylin Os Manager</source>
+        <translation>ကိုင်လင်ဂါစီရင်</translation>
+    </message>
+    <message>
+        <location filename="../disk_monitor_exception_popup.cpp" line="54"/>
+        <source>Not prompted within 3 days</source>
+        <translation>နောက် ၃ ရက်အတွင်း ထပ်မပြပါနှင့်</translation>
+    </message>
+    <message>
+        <location filename="../disk_monitor_exception_popup.cpp" line="66"/>
+        <source>Cancel</source>
+        <translation>ပယ်ဖျက်ရန်</translation>
+    </message>
+    <message>
+        <location filename="../disk_monitor_exception_popup.cpp" line="73"/>
+        <source>Clean up immediately</source>
+        <translation>ချက်ခြင်း ရှင်းလင်းရန်</translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <location filename="../driverinstall/aptinstaller.cpp" line="68"/>
+        <source>Install timeout.</source>
+        <translation>ထည့်သွင်းခြင်း အချိန်ကုန်ပြီးပါပြီ။</translation>
+    </message>
+</context>
+<context>
+    <name>ServiceSupportPopup</name>
+    <message>
+        <location filename="../service_support_popup.cpp" line="22"/>
+        <source>Kylin Os Manager</source>
+        <translation>ကိုင်လင်ဂါစီရင်</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="40"/>
+        <source>Detected system abnormality, please contact the technical service immediately or call 400!</source>
+        <translation>စနစ်မှာ ပြဿနာတစ်ခုရှိနေသည်ကို တွေ့ရှိပါသည်၊ ချက်ခြင်း နည်းပညာပံ့ပိုးမှုကိုဆက်သွယ်ပါ သို့မဟုတ် ၄၀၀ ဖုန်းနံပါတ်ကိုခေါ်ဆိုပါ။</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="57"/>
+        <source>Or contact us</source>
+        <translation>သို့မဟုတ် ကျွန်ုပ်တို့ကိုဆက်သွယ်ပါ</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="61"/>
+        <source>Telephone</source>
+        <translation>ဖုန်း</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="67"/>
+        <source>Mail</source>
+        <translation>အီးမေးလ်</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="89"/>
+        <source>For common questions, please refer to</source>
+        <translation>အများအားဖြင့်တွေ့ရသည့် ပြဿနာများအတွက် ကျေးဇူးပြု၍ ဖတ်ရှုပါ</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="92"/>
+        <source>KylinOs offical website</source>
+        <translation>ကိုင်လင်ဂါဆော့ဖ်ဝဲ ရုံးသုံးဝက်ဘ်ဆိုက်</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="133"/>
+        <source>Cancel</source>
+        <translation>ပယ်ဖျက်ရန်</translation>
+    </message>
+    <message>
+        <location filename="../service_support_popup.cpp" line="140"/>
+        <source>Clean</source>
+        <translation>ရှင်းလင်းရန်</translation>
+    </message>
+</context>
+<context>
+    <name>SessionClient</name>
+    <message>
+        <location filename="../sessionclient.cpp" line="126"/>
+        <location filename="../sessionclient.cpp" line="175"/>
+        <location filename="../sessionclient.cpp" line="221"/>
+        <location filename="../sessionclient.cpp" line="275"/>
+        <location filename="../sessionclient.cpp" line="357"/>
+        <location filename="../sessionclient.cpp" line="360"/>
+        <location filename="../sessionclient.cpp" line="470"/>
+        <location filename="../sessionclient.cpp" line="473"/>
+        <source>Kylin Os Manager</source>
+        <translation>ကိုင်လင်ဂါစီရင်</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="129"/>
+        <location filename="../sessionclient.cpp" line="178"/>
+        <location filename="../sessionclient.cpp" line="224"/>
+        <location filename="../sessionclient.cpp" line="278"/>
+        <source>Device Manager</source>
+        <translation>ကိရိယာစီမံခန့်ခွဲမှု</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="130"/>
+        <source>The system detects that a new device has an installable driver. Do you want to install?</source>
+        <translation>အသစ်ထည့်သွင်းနိုင်သည့် driver ကိရိယာတစ်ခု တွေ့ရှိပါသည်၊ driver ကိုထည့်သွင်းမလား။</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="131"/>
+        <source>Close</source>
+        <translation>ပယ်ဖျက်ရန်</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="131"/>
+        <source>Install</source>
+        <translation>ထည့်သွင်းရန်</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="179"/>
+        <source>Start installing the corresponding driver</source>
+        <translation>ဆိုင်းကားကို ထည့်သွင်းခြင်း စတင်ပါ</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="225"/>
+        <source>Driver Installed</source>
+        <translation>driver ကို ထည့်သွင်းပြီးပါပြီ။</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="227"/>
+        <source>Open Scanner</source>
+        <translation>စကင်ဖတ်ခြင်း ဖွင့်ရန်</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="279"/>
+        <source>Driver install failed.</source>
+        <translation>driver ထည့်သွင်းခြင်း မအောင်မြင်ပါ။</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="361"/>
+        <source>Detected log files that are too large, it is recommended to clean them up</source>
+        <translation>log ဖိုင်များ အရွယ်အစားကြီးနေသည်ကို တွေ့ရှိပါသည်၊ ရှင်းလင်းရန်အကြံပြုအပ်ပါသည်။</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="364"/>
+        <location filename="../sessionclient.cpp" line="477"/>
+        <source>Cancel</source>
+        <translation>ပယ်ဖျက်ရန်</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="366"/>
+        <location filename="../sessionclient.cpp" line="479"/>
+        <source>One click cleaning</source>
+        <translation>တစ်ချက်နှိပ်၍ ရှင်းလင်းရန်</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="398"/>
+        <source>Detected log files that are too large, please go to Kylin Os Manager for cleaning!</source>
+        <translation>log ဖိုင်များ အရွယ်အစားကြီးနေသည်ကို တွေ့ရှိပါသည်၊ ကိုင်လင်ဂါစီရင်သို့သွား၍ ရှင်းလင်းပေးပါ။</translation>
+    </message>
+    <message>
+        <location filename="../sessionclient.cpp" line="474"/>
+        <source>The avaliable disk space is less than %1%, please clean it up in a timely manner</source>
+        <translation>disk တွင် နေရာလွတ် %1% ထက်နည်းနေသည်၊ ချက်ခြင်း ရှင်းလင်းပေးပါ။</translation>
+    </message>
+</context>
+</TS>
